src/main.ts

```diff
@@ -163,14 +164,19 @@ function createSourceReplacingCompilerHost(
  * Compiles TypeScript code into Closure-compiler-ready JS.
  * Doesn't write any files to disk; all JS content is returned in a map.
```

Comment:
comment on fileNames vs files function API. I find it a bit odd to be able to pass both.